config/settings/base.py

```diff
@@ -203,3 +196,21 @@
 RABBITMQ_MANAGEMENT_INTERFACE = env.str("RABBITMQ_MANAGEMENT_INTERFACE", "")
 
 PROMETHEUS_EXPORT_MIGRATIONS = env.bool("PROMETHEUS_EXPORT_MIGRATIONS", False)
+SENTRY_DSN = env.str("SENTRY_DSN", "")
```

Comment:
I see the previous env was called RAVEN_DSN, can we fall back to that if SENTRY_DSN is not present

Reply:
Can we change to this?
`SENTRY_DSN = env.str("SENTRY_DSN", env.str("RAVEN_DSN", ""))`